File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-Jamba-Android-v2.4.1/Android_Raleys/LibAndroid/src/com/raleys/libandroid/WebServiceError.java
package com.raleys.libandroid;


public class WebServiceError
{
	public int errorCode;
	public String errorMessage;

	public WebServiceError()
	{
	}


	public WebServiceError(int code, String message)
	{
		errorCode = code;
		errorMessage = message;
	}
}

File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-jamba-development/iOS_Raleys/LibiOS/LibiOS/Classes/WebServiceError.m
//
//  WebServiceError.m
//  LibiOS
//
//  Created by Bill Lewis on 10/10/13.
//  Copyright (c) 2013 Raleys. All rights reserved.
//

#import "WebServiceError.h"

@implementation WebServiceError

@synthesize errorCode;
@synthesize errorMessage;


- (id)initWithErrorCode:(int)code andMessage:(NSString *)message
{
    self = [super init];

    if(self)
    {
        errorCode = code;
        errorMessage = message;
    }

    return self;
}
@end
